src/Client/Updater.php

```diff
@@ -174,59 +174,62 @@ public function refresh(bool $force = false): bool
 
         $nowDate = $this->getCurrentTime();
 
-        // TUF-SPEC-v1.0.9 Section 5.1.11. Will be used in spec step 5.4.3.
+        // TUF-SPEC-v1.0.10 Section 5.1.11. Will be used in spec step 5.4.?.
```

Comment:
5.4.? is not a step :) 

If we were to add support for consistent snapshots, this would be used in 5.3, 5.4, and 5.5.2. In the meantime, we don't support it and throw an error in supportsConsistentSnapshots(). For clarity, let's have a follow-up to replace this comment and commented-out code line with a @todo to an issue to add support for consistent snapshots.